CMakeLists.txt

```
@@ -47,6 +47,7 @@ tr_auto_option(ENABLE_GTK "Build GTK client" AUTO)
 tr_auto_option(ENABLE_QT "Build Qt client" AUTO)
 tr_auto_option(ENABLE_MAC "Build Mac client" AUTO)
 tr_auto_option(REBUILD_WEB "Rebuild the web client's generated assets. Requires Node.js and network access." OFF)
+tr_auto_option(INSTALL_WEB "Install the web client's generated assets." ON)
```

There's no `AUTO` handling (nor is it necessary).
```suggestion
option(INSTALL_WEB "Install the web client's generated assets." ON)
```